src/main/java/com/google/gcloud/storage/StorageService.java

```diff
@@ -412,14 +420,14 @@ public static Builder builder() {
   Blob create(Blob blob, byte[] content, BlobTargetOption... options);
 
   /**
-   * Return the requested bucket.
+   * Return the requested bucket or {@code null} if not found.
```

Comment:
Other Java APIs throw FileNotFoundEx or NoSuchFileEx when the file does not exist. I think it would better to do the same thing here.